indexer/schemas/clickhouse/bsc_block.sql

```diff
@@ -1,8 +1,9 @@
-CREATE TABLE IF NOT EXISTS bsc_block {
+CREATE TABLE IF NOT EXISTS bsc_block 
+(
     number UInt64,
     hash FixedString(66),
     parent_hash FixedString(66),
-    nonce UInt256,
+    nonce String,
```

Comment:
Why different?

Reply:
int() cannot convert hex string. Ex: ValueError: invalid literal for int() with base 10: '0x0000000000000000'